trashtalk/seeclickfix.py

```diff
@@ -2,12 +2,17 @@
 
 import requests
 from input_handling import twelve_hour_time
+from trashtalk.master_keys import MasterKeys
+key_chain = MasterKeys()
```

Comment:
Same as above. We are using environment variables currently to handle keys. But if there was some other purpose you had in mind can you describe it?